launch/mono_camera.launch

```diff
@@ -3,6 +3,7 @@
   <arg name="ip" default="" doc="The IP for the camera to connect to"/>
   <arg name="guid" default="50-0503343290" doc="The GUID for the camera to connect to"/>
   <arg name="frame_id" default="left_optical" doc="The frame id of the camera"/>
+  <arg name="use_measurement_time" default="false" doc="Use the measurement time from the camera"/>
```

Comment:
Is there a offset in between the measurement time and the wall clock?

Reply:
The Mako cameras use TAI time instead of UTC, so a `ptp_offset` param was created to account for the difference.

Reply:
How do we find this ptp_offset?

Reply:
Difference between UTC and TAI is currently 37 seconds:
http://leapsecond.com/java/gpsclock.htm
The parameter defaults to -37 at the moment.